_posts/2018-12-22-35c3.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Void Linux at 35c3"
+comments: true
+---
+
+![35C3 Void](/assets/img/35c3-void.png)
+
+2018 has been an intense year for Void. There were many ups and downs. To wrap
+up the months behind we're meeting at the [35c3](https://events.ccc.de/category/congress/35c3/).
```

Comment:
maybe swap this for "To wrap up the year we're meeting at..." as that reads a bit more naturally.